src/content/download.mdx

```diff
@@ -42,16 +42,16 @@ yarn add zeppelin-element-library
 Use the cached version of our files from our CDN
 
 ```js
-<link rel="stylesheet" href="https://unpkg.com/zeppelin-element-library@latest/bundle/zeppelin-element-library.css" >
-<script src="https://unpkg.com/zeppelin-element-library@latest/bundle/zeppelin-element-library.min.js" ></script>
+<link rel="stylesheet" href="https://unpkg.com/zeppelin-element-library@latest/bundle/zel.css" >
```

Comment:
Wrong link, there is no bundle folder anymore. It should point to "https://unpkg.com/zeppelin-element-library@1.3.1/zel.css"